### PR TITLE
Release 1.0-rc1

### PR DESCRIPTION
**Pull request by notriddle**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -763,10 +763,7 @@ impl Document {
     }
 
     fn serialize_opts() -> SerializeOpts {
-        SerializeOpts {
-            traversal_scope: TraversalScope::ChildrenOnly,
```
Comment (Eijebong) [inline src/lib.rs]: Is this the default now ?

Comment (lnicola) [reply src/lib.rs]: Looks like it was, even before: https://github.com/servo/html5ever/commit/015cfb029be4d6404ec0cc3e68186cb82bd47db6.

**Comment by lnicola** [comment]
We should hold this until the other issues get closed.

**Comment by notriddle** [comment]
The two additional URL resolvers? Those can be added without breaking changes.

Wait, oops, I need to make sure we can add new URL resolvers without them being breaking changes. I'll open a PR to fix that.

**Comment by lnicola** [comment]
I wanted to open an issue for adding a hidden variant (`#[non_exhaustive] will be available soon) to that enum, but it seemed overkill.

**Comment by notriddle** [comment]
Opened #51

**Comment by lnicola** [comment]
We could also have an `-rc` and post that on Reddit and https://users.rust-lang.org/, just in case anyone finds an issue with the API.

**Comment by notriddle** [comment]
Sure, let's do that.

**Comment by notriddle** [comment]
Fixed the html_root_url.

**Comment by notriddle** [comment]
Now that #31 is done, I think we can make the RC1 release. Sure, there's still some nice-to-have features on the backlog, but there's no reason we can't make feature-releases after 1.0 (they just have to not break existing APIs, of course).

**Comment by lnicola** [comment]
Just one concern, does #48 obsolete `UrlRelative::RewriteWithBase`?

**Comment by Eijebong** [comment]
@lnicola If it's the case we can deprecate `UrlRelative::RewriteWithBase` when #48 is done and remove it in the 2.0 release.

**Comment by lnicola** [comment]
This will probably need to be rebased.

**Comment by lnicola** [comment]
bors r= @Eijebong , @notriddle , @lnicola

**Comment by lnicola** [comment]
@notriddle I made a draft release, so if you want to hit "Publish"..

**Comment by notriddle** [comment]
Done.

**Comment by lnicola** [comment]
Still shows up as draft for me.

**Comment by notriddle** [comment]
Okay, now the crates is published on crates.io *and* the release is published on GitHub.

**Comment by notriddle** [comment]
Also, there's now a note about it in ["What are you working on this week" on u.rl.o](https://users.rust-lang.org/t/whats-everything-working-on-this-week-39-2017/13029/8).

**Comment by notriddle** [comment]
Anybody else want to submit it to Reddit?

**Comment by lnicola** [comment]
Submitted here: https://www.reddit.com/r/rust/comments/72kp3c/ammonia_100rc1_an_html_sanitization_crate/

**Pull request merged**